### DISCUSSION TITLE
PIMCore version upgrade from 4.2 to 5

### DISCUSSION THREAD

**Discussion by yadavab2112** [pimcore/pimcore#10678] [General]
Hi All, 
We are using pimcore 4.2 , So we need to upgrade to version 5 from 4.2. 
So can we get any documentation for the same as we are having lots of error?

Thanks
Best Regards
Abhishek

**Comment by dvesh3** [comment]
please see https://pimcore.com/docs/pimcore/5.8/Development_Documentation/Installation_and_Upgrade/Updating_Pimcore/Upgrade_from_4_to_5/index.html

**Reply by yadavab2112** [reply]
This is the error we are getting :-

An error occurred when executing the ""assets:install --symlink web"" command:



Could not open input file: app/console

**Reply by dvesh3** [reply]
Sorry, Pimcore 5 is not supported anymore. Please see supported versions here https://github.com/pimcore/pimcore#supported-versions

**Reply by yadavab2112** [reply]
So can we upgrade to higher version from 4.6 directly please suggest
Thanks!!

**Comment by frithjof** [comment]
best way would be to make a new web based on pimcore 10 and manually migrate data or write custom migration scripts

**Comment by brusch** [comment]
See also: https://github.com/pimcore/pimcore/discussions/9628
